src/tabs/Course__c.tab

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomTab xmlns="http://soap.sforce.com/2006/04/metadata">
     <customObject>true</customObject>
-    <mobileReady>false</mobileReady>
```

Comment:
@m-kinkade @skosgi1 whats the reason for this to reenable, disabled  mobile functionality?

Reply:
@jofsky looks like that is not supported in api 48.0 https://success.salesforce.com/answers?id=9063A000000a5FKQAY